examples/foomedical/src/pages/health-record/Response.tsx

```diff
@@ -0,0 +1,79 @@
+import { Box, Paper, Stack, Title, TitleOrder } from '@mantine/core';
+import { formatDate } from '@medplum/core';
+import { QuestionnaireResponseItem, QuestionnaireResponseItemAnswer } from '@medplum/fhirtypes';
+import { CodeableConceptDisplay, QuantityDisplay, RangeDisplay, useMedplum } from '@medplum/react';
+import { useParams } from 'react-router-dom';
+
+export function Response(): JSX.Element {
+  const medplum = useMedplum();
+  const { responseId } = useParams();
+  const questionnaireResponse = medplum.searchOne('QuestionnaireResponse', `_id=${responseId}`).read();
```

Comment:
Not using `useResource` here to keep it consistent with other health record pages and to make use of the `<Suspense />` loading in the root page. LMK if you'd prefer to keep it this way or update to the new standard.